app/gui2/src/components/GraphEditor/widgets/WidgetSelection.vue

```diff
@@ -129,7 +139,11 @@ function toggleDropdownWidget() {
 }
 
 function onClick(index: number, keepOpen: boolean) {
-  selectedIndex.value = index
+  if (index < customTags.value.length) {
+    customTags.value[index]!.onClick!()
```

Comment:
`tagFromCustomItem` could return a subtype of `Tag` with an `onClick` property, so we would not need the second non-null assertion here.

Reply:
Thanks, it’s clever